components/supervisor/frontend/src/index.ts

```diff
@@ -8,7 +8,12 @@
  * <script type="text/javascript" src="/_supervisor/frontend/main.js" charset="utf-8"></script> should be inserted to index.html as first body script,
  * all other IDE scripts should go afterwards, head element should not have scripts
  */
-
+import { IDEMetricsServiceClient } from "./ide/ide-metrics-service-client";
```

Comment:
@iQQBot can we add another metric: `gitpod_supervisor_frontend_window_total` to count each window with supervisor frontend

We will measure then rate of errors against rate of loaded windows. 🙏

Reply:
yes increase once at the top of supervisor frontend loading